src/benchmarks/averageFloat32x4Load.js

```diff
@@ -19,7 +19,6 @@
 
   // Benchmark data, initialization and kernel functions
   var a   = new Float32Array(10000);
-  var ax4 = new Float32x4Array(a.buffer);
```

Comment:
Actually, I think we can remove this test case which is covered by averageFloat32x4.js already. Your thoughts?